### DISCUSSION TITLE
Bagaimana cara membuat Tag Conditional untuk url/page tertentu di Mecha CMS?

### DISCUSSION THREAD

**Discussion by bungfrangki** [mecha-cms/mecha#274] [Q&A] [answered]
Maaf, mas Taufik.
saya ingin mendokumentasikan pertanyaan sy dan jawaban mas Taufik di jalur perpesanan lain ke forum ini. Barangkali bisa berguna buat saya kedepan dan juga buat pengguna/calon pengguna Mecha CMS lainnya.

Terkait pertanyaan, 
saya awalnya bikin tag kondisi seperti di bawah ini untuk 

<pre><code>&lt;?php if ($site-&gt;has(&#039;page&#039;, array(&#039;contact&#039;,&#039;about&#039;)) !== false): ?&gt;</code></pre>

Dan sudah bisa ditebak, tag kondisinya salah alias tidak berfungsi.

yang benar adalah seperti ini (jawaban dari mas Taufik):
<pre><code>&lt;?php if ($site-&gt;is('page') && ($url-&gt;path === '/about' || $url-&gt;path === '/contact')): ?&gt;</code></pre>

atau seperti ini:
<pre><code>&lt;?php if ($site-&gt;is('page') && in_array($url-&gt;path, ['/about', '/contact'])): ?&gt;</code></pre>

**Comment by taufik-nurrohman** [comment, marked as answer]
Siap. Terima kasih sudah berbagi 👍
